### DISCUSSION TITLE
Feature Request: qoute and reply to thread feature

### DISCUSSION THREAD

**Discussion by MBoronNeotreks** [ankurk91/google-chat-electron#61] [Q&A]
It would be useful to have an option to **reply** to a particular message.
This feature is not in the Google Chat but there are some _workarounds_ - discussed e.g. here: https://allthings.how/quote-reply-google-chat/

It would be nice if such reply feature would be part of the Electron app.

Thank you.

**Comment by ankurk91** [comment]
i have no plans to add chrome extensions, since chrome extension is experimental in electron

**Reply by MBoronNeotreks** [reply]
And could the Electron app implement this reply feature itself (without the Chrome extension)?
Just copy the selected message, surround it with **```**, and paste it into the message text-box?

**Reply by aaronabl** [reply]
I agree with this. I think the feature itself is pretty easy to implement and would make the app loads better.

I like using the electron app as it is faster, but also because I was running a Chrome Profile that had multiple Google Accounts. If I create a profile specifically for work, the Chrome "app" works as well. But I guess it does slow down a fair bit compared to the electron app.

Something to consider?
